### PR TITLE
Add globalization

### PR DESCRIPTION
**Pull request by superkhau**
Connect to strongloop/loopback/issues/2422

### CODE REVIEW

**Comment by superkhau** [comment]
@slnode test please

File: lib/sqlite3.js [hunk 1]
```diff
@@ -372,7 +374,7 @@ function _convertBoolean(value) {
     return 1;
   if (booleanFalse.indexOf(value) !== -1)
     return 0;
-  return Error('SQLITE3: Invalid boolean default: ' + value);
+  return Error(g.f('SQLITE3: Invalid boolean default: %s', value));
```
Comment (0candy) [inline lib/sqlite3.js]: Add `{{}}` for `{{SQLITE3}}`

File: lib/sqlite3.js [hunk 2]
```diff
@@ -520,7 +522,7 @@ SQLite3.prototype.toColumnValue = function(property, value) {
   switch (property.type.name) {
     case 'Number':
       if (isNaN(value))
-        return new InvalidParam('SQLITE3: Invalid number: ' + value);
+        return new InvalidParam(g.f('SQLITE3: Invalid number: %s', value));
```
Comment (0candy) [inline lib/sqlite3.js]: Add {{}} for {{SQLITE3}}

**Comment by superkhau** [comment]
@0candy Updated, review again pls.

File: lib/sqlite3.js [hunk 3]
```diff
@@ -372,7 +374,7 @@ function _convertBoolean(value) {
     return 1;
   if (booleanFalse.indexOf(value) !== -1)
     return 0;
-  return Error('SQLITE3: Invalid boolean default: ' + value);
+  return Error(g.f('{{SQLITE3}}: Invalid boolean default: %s', value));
```
Comment (0candy) [inline lib/sqlite3.js]: You have to run slt-globalize -e again for intl/en/messages.json to get updated with the brackets change.

**Comment by superkhau** [comment]
@0candy Updated, PTAL again.

**Comment by 0candy** [comment]
Please make sure `var SG = require('strong-globalize');` is at the beginning of the file.

**Comment by superkhau** [comment]
@slnode test please

**Comment by superkhau** [comment]
@0candy @loay PTAL, updated with latest conventions.

**Comment by 0candy** [comment]
LGTM

File: lib/migration.js [hunk 4]
```diff
@@ -30,7 +32,7 @@ function mixinMigration(SQLite3) {
     async.eachSeries(models, function(model, done) {
       if (!(model in self._models)) {
         return process.nextTick(function() {
-          done(new Error('Model not found: ' + model));
+          done(new Error(g.f('Model not found: %s', model)));
```
Comment (loay) [inline lib/migration.js]: {{Model}}

**Comment by loay** [comment]
LGTM

**Comment by superkhau** [comment]
Unrelated CI failure. Merging.

**Pull request merged**